Update introducing-metrics-in-innersource.md

arlou opened the pull request:
I consolidated some metrics candidates into the pattern from forums and InnerSource Commons slack channel. Hope that helps.

arlou committed:
Update introducing-metrics-in-innersource.md

spier commented on patterns/1-initial/introducing-metrics-in-innersource.md:
```diff
@@ -89,8 +100,11 @@ Initial
 - Georg
 - Bob
+- Aaron Steward
+- Willson Mar
```
```suggestion
- Wilson Mar
```

Possibly spelled like this?

Also did you check with both Aaron and Wilson that they would like to be mentioned here?

spier commented on patterns/1-initial/introducing-metrics-in-innersource.md:
```diff
@@ -64,7 +64,18 @@ Produce a characterization of metrics as this might be helpful for others to und
 * A third class of metrics are KPIs that we believe will improve the raw amount of InnerSourcing happening (e.g. mean-time-to-review, automated test runs on PRs, etc.).
 
-Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process.
+Note: The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. Below is the list consolidating some metrics candidates by business goals.
```
```suggestion
_Note:_ The aforementioned metrics are some common examples. They aren't a one-size-fits-all. We expect the business goals of the organization to drive the metric selection process. 

The list below consolidates some metrics candidates, grouped by business goals.

```

spier commented on patterns/1-initial/introducing-metrics-in-innersource.md:
```diff
@@ -64,7 +64,18 @@ Produce a characterization of metrics as this might be helpful for others to und
+| Innovation |  |
+| Others | # of repositories using InnerSource components (managed contributing guidelines, assigned TCs)<br>Days of host's responsiveness on the issues/PRs created by external contributors |
+
 
```
```suggestion

```

Fix linter issue

spier commented on patterns/1-initial/introducing-metrics-in-innersource.md:
```diff
@@ -64,7 +64,18 @@ Produce a characterization of metrics as this might be helpful for others to und
+| Business Goals | Metric Candidates |
+|---|---|
+| Remove silos and bottlenecks | # of PRs from external contributors<br># of PRs from external contributors that are merged<br># of outside collaborators<br># of issues created from external contributors<br># of reviews from external contributors |
+| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests (proposed by Jack) |
```
Do you want to keep this "(proposed by Jack)" note in here?

We don't have to mention where which metric comes from. It is actually quite likely that many organizations have discover the same metrics over and over again anyways :)

arlou replied:
Sorry, that's typo. I should remove it from the sentence

arlou replied:
BTW, I will contact Wilson and Aaron to check if they concern on being mentioned in the page; Thanks for @spier 's review and reminding

arlou replied:
Confirmed no concern from Aaron, and waiting for Wilson's confirmation

arlou replied:
Update, just got reply from Wilson. Confirmed he has no concern on this.

spier reviewed (changes requested):
Great idea to add metric candidates @arlou! Thanks for this.

What would you think about adding the links to the public forums/websites where you found some of this information, in case people want to read more about this. We cannot reference the conversations in the ISC Slack (as they are under Chatham House Rule) but we can reference anything that's already public.

spier committed:
Fix linter issue

spier commented on patterns/1-initial/introducing-metrics-in-innersource.md:
```diff
@@ -64,8 +64,17 @@ Produce a characterization of metrics as this might be helpful for others to und
+| Improve quality | # of automated test runs on PRs<br># of issues created from external contributors |
+| Innovation |  |
+| Others | # of repositories using InnerSource components (managed contributing guidelines, assigned TCs)<br>Days of host's responsiveness on the issues/PRs created by external contributors |
 ## Resulting Context
```
```suggestion

## Resulting Context
```

spier committed:
Fix linter error

arlou committed:
Update patterns/1-initial/introducing-metrics-in-innersource.md

Co-authored-by: Sebastian Spier <github@spier.hu>

arlou committed:
Update patterns/1-initial/introducing-metrics-in-innersource.md

Co-authored-by: Sebastian Spier <github@spier.hu>

arlou committed:
Update introducing-metrics-in-innersource.md

arlou committed:
Update the link to the public page I referred to

Hi reviewer, I've added the link here, but I'm sure if placing the link on the name is good or not. Please feel free to adjust or let me do it if needed. Thanks.
(Actually, this is my first time contributing to public repositories; please let me know if I violate some rules.)

arlou committed:
fixing patterns/1-initial/introducing-metrics-in-innersource.md:67:182

arlou committed:
type Aaron Steward -> Aaron Stewart

arlou committed:
Change "outside collaborators" to "external contributors"

For consistency in content

spier commented on patterns/1-initial/introducing-metrics-in-innersource.md:
```diff
@@ -64,7 +64,20 @@ Produce a characterization of metrics as this might be helpful for others to und
+| Knowledge sharing | # of clone or forks from external guests<br># of visits from external guests |
+| Creating reusable software | # of submodule usage<br># of code reuse across projects<br># of clone or forks from external guests |
+| Increase development speed | Time to market<br>Mean-time-to-review |
+| Employee satisfication& talent retention |  |
```
```suggestion
| Employee satisfaction & talent retention |  |
```

Fix typo and whitespace

spier committed:
Fix typo and whitespace

spier commented on patterns/1-initial/introducing-metrics-in-innersource.md:
```diff
@@ -89,8 +102,11 @@ Initial
 - Georg
 - Bob
+- [Aaron Stewart](https://github.com/a-a-ron/innersource-template-pluralsight/tree/master/metrics)
+- Wilson Mar
```
@arlou do you have any online resource by Wilson Mar that you want to reference here?

arlou replied:
hi @spier, thanks for checking; So far, no referred link on Wilson as I referred to his inputs from Slack channel.

spier reviewed (approved):
LGTM
